quicklogic/common/utils/arch_import.py

```diff
@@ -388,7 +387,7 @@ def main():
     with open(args.vpr_db, "rb") as fp:
         db = pickle.load(fp)
 
-        cells_library = db["cells_library"]
+        # cells_library = db["cells_library"]
```

Comment:
I presume that the linter indicates that the variable is not used anywhere. You may as well remove such lines completely, here and everywhere else in the code.